ecdsa/Cargo.toml

```diff
@@ -2,25 +2,25 @@
 name    = "ecdsa"
 version = "0.13.0-pre" # Also update html_root_url in lib.rs when bumping this
 description = """
-Signature and elliptic curve types providing interoperable support for the
-Elliptic Curve Digital Signature Algorithm (ECDSA)
+Pure Rust implementation of the Elliptic Curve Digital Signature Algorithm
+(ECDSA) as specified in FIPS 186-4 (Digital Signature Standard)
 """
 authors = ["RustCrypto Developers"]
 license = "Apache-2.0 OR MIT"
-repository = "https://github.com/RustCrypto/signatures"
+repository = "https://github.com/RustCrypto/signatures/tree/master/ecdsa"
 readme = "README.md"
 categories = ["cryptography", "no-std"]
 keywords = ["crypto", "ecc", "nist", "secp256k1", "signature"]
 edition = "2021"
 rust-version = "1.56"
 
 [dependencies]
-elliptic-curve = { version = "0.11", default-features = false, features = ["sec1"] }
-hmac = { version = "0.11", optional = true, default-features = false }
-signature = { version = ">= 1.3.1", default-features = false, features = ["rand-preview"] }
+elliptic-curve = { version = "0.11.1", default-features = false, features = ["sec1"] }
+signature = { version = ">= 1.3.1, <1.5", default-features = false, features = ["rand-preview"] }
```

Comment:
What's the reason for adding the <1.5 restriction here? This makes it impossible to use ecdsa 0.13 and ecdsa 0.14 dependencies in one project:

```
➜  webb-rs git:(main) cargo check
    Blocking waiting for file lock on package cache
    Updating crates.io index
error: failed to select a version for `signature`.
    ... required by package `ecdsa v0.13.3`
    ... which satisfies dependency `ecdsa-core = "^0.13"` of package `k256 v0.10.4`
    ... which satisfies dependency `k256 = "^0.10.4"` of package `cosmwasm-crypto v1.0.0`
    ... which satisfies dependency `cosmwasm-crypto = "^1.0.0"` of package `cosmwasm-std v1.0.0`
    ... which satisfies dependency `cosmwasm-std = "^1.0.0"` of package `webb-proposals v0.4.4 (/Users/simon/projects/CosmWasm/webb-rs/proposals)`
versions that meet the requirements `>=1.3.1, <1.5` are: 1.4.0, 1.3.2, 1.3.1

all possible versions conflict with previously selected packages.

  previously selected package `signature v1.5.0`
    ... which satisfies dependency `signature = "^1.5"` (locked to 1.5.0) of package `ecdsa v0.14.3`
    ... which satisfies dependency `ecdsa-core = "^0.14"` (locked to 0.14.3) of package `k256 v0.11.3`
    ... which satisfies dependency `k256 = "^0.11"` (locked to 0.11.3) of package `coins-bip32 v0.7.0`
    ... which satisfies dependency `coins-bip32 = "^0.7.0"` (locked to 0.7.0) of package `coins-bip39 v0.7.0`
    ... which satisfies dependency `coins-bip39 = "^0.7.0"` (locked to 0.7.0) of package `ethers-signers v0.17.0`
    ... which satisfies dependency `ethers-signers = "^0.17.0"` (locked to 0.17.0) of package `ethers v0.17.0`
    ... which satisfies dependency `ethers = "^0.17.0"` (locked to 0.17.0) of package `webb v0.5.0 ([...]/CosmWasm/webb-rs)`

failed to select a version for `signature` which could resolve this conflict
```

Is there any practical way around that?

Reply:
See this issue: https://github.com/RustCrypto/utils/issues/723

`rfc6979` v0.3.0 does not have this restriction.